ui/user/src/routes/v2/admin/audit-logs/+page.svelte

```diff
@@ -1,95 +1,164 @@
 <script lang="ts">
-	import { browser } from '$app/environment';
+	import { fade, slide } from 'svelte/transition';
+	import { X, ChevronLeft, ChevronRight } from 'lucide-svelte';
+	import { throttle } from 'es-toolkit';
+	import { page } from '$app/state';
 	import { afterNavigate, goto } from '$app/navigation';
-	import AuditDetails from '$lib/components/admin/audit-logs/AuditDetails.svelte';
-	import Calendar, { type DateRange } from '$lib/components/Calendar.svelte';
+	import { type DateRange } from '$lib/components/Calendar.svelte';
 	import Layout from '$lib/components/Layout.svelte';
+	import Search from '$lib/components/Search.svelte';
 	import { PAGE_TRANSITION_DURATION } from '$lib/constants';
-	import { type OrgUser, type AuditLogFilters, AdminService } from '$lib/services';
-	import { formatTimeRange, getTimeRangeShorthand } from '$lib/time';
-	import { Captions, X } from 'lucide-svelte';
-	import { fade } from 'svelte/transition';
+	import { type OrgUser, type AuditLogFilters, AdminService, type AuditLog } from '$lib/services';
+	import { getUser, type PaginatedResponse } from '$lib/services/admin/operations';
+	import { clickOutside } from '$lib/actions/clickoutside';
+	import { dialogAnimation } from '$lib/actions/dialogAnimation';
+	import AuditLogDetails from '$lib/components/admin/audit-logs/AuditLogDetails.svelte';
+	import AuditFilters from '$lib/components/admin/audit-logs/AuditFilters.svelte';
+	import { tooltip } from '$lib/actions/tooltip.svelte';
+	import AuditLogsTable from './AuditLogs.svelte';
+	import AuditLogsTimeline from './AuditLogsTimeline.svelte';
+	import AuditLogCalendar from './AuditLogCalendar.svelte';
+	import { endOfDay, set } from 'date-fns';
 
 	const duration = PAGE_TRANSITION_DURATION;
 
-	let users = $state<OrgUser[]>([]);
-	let currentFilters = $state<AuditLogFilters & { mcpId?: string | null }>({});
-
-	let timeRange = $derived(
-		currentFilters.startTime || currentFilters.endTime
-			? {
-					startTime: currentFilters.startTime ?? '',
-					endTime: currentFilters.endTime ?? ''
-				}
-			: {
-					startTime: new Date(Date.now() - 7 * 24 * 60 * 60 * 1000).toISOString(),
-					endTime: new Date().toISOString()
-				}
+	let auditLogsResponse = $state<PaginatedResponse<AuditLog>>();
+	const auditLogsTotalItems = $derived(auditLogsResponse?.total ?? 0);
+
+	let pageIndex = $state(0);
+	const pageLimit = $state(40);
+
+	const numberOfPages = $derived(Math.ceil(auditLogsTotalItems / pageLimit));
+	const pageOffset = $derived(pageIndex * pageLimit);
+
+	const remoteAuditLogs = $derived(
+		(auditLogsResponse?.items ?? []).map(({ createdAt, ...restProps }) => ({
+			...restProps,
+			createdAt: new Date(createdAt.slice(0, -6)) + 'Z'
+		}))
 	);
```

Comment:
@ryu-man When I change these lines back to thing, everything works as expected for me:
```suggestion
	const remoteAuditLogs = $derived(auditLogsResponse?.items ?? []);
```

<img width="1445" height="1227" alt="Image" src="https://github.com/user-attachments/assets/ced251fe-5fa1-4b22-9975-7fe8bcd60da4" />

Reply:
Here is a better example where I am filtering down to a single minute:
<img width="1339" height="814" alt="Image" src="https://github.com/user-attachments/assets/678cfcab-7df5-46ca-8a72-379cb0df3524" />